### PR TITLE
Ensure group selection box disabled properly

### PR DESCRIPTION
**Pull request by acron0**
**Ensure group selection box disabled properly**
Also resets on edit

### CODE REVIEW

File: src/cljs/witan/ui/components/data.cljs [hunk 1]
```diff
@@ -887,8 +887,11 @@
                 :content @groups}]
               [:h3 (get-string :string/share-data-with)]
               [shared/group-dropdown {:groups expanded-user-groups
-                                      :current-group selected-group}
-               #(reset! selected-group %)]
+                                      :current-group selected-group
+                                      :disabled? pending?}
+               #(do
+                  (reset! post-send? false)
+                  (reset! selected-group %))]
```
Comment (otfrom) [inline src/cljs/witan/ui/components/data.cljs:894]: what will % be here?

**Review by otfrom — APPROVED** [review]
A question, but lgtm

**Review by mattford63 — APPROVED** [review]

File: src/cljs/witan/ui/components/data.cljs (hunk 1 above)
Comment (acron0) [reply src/cljs/witan/ui/components/data.cljs:894]: It's a group object. Name, id etc

Comment (acron0) [reply src/cljs/witan/ui/components/data.cljs:894]: All of the shared components return context appropriate values, rather than js objects/events

**Pull request merged**